joomla/joomla-cms
# [Isis] Add styling to com_template file tree

Pull Request for Issue # .

### Summary of Changes
Adds styling to com_template file tree. If favourable the same styling can be added to other trees.

### Testing Instructions
Apply patch and navigate to Extensions -> Templates -> Templates -> [Protostar]. Check file tree.

### Before Patch
![template-tree1](https://cloud.githubusercontent.com/assets/2803503/22886577/70d7d75a-f1f6-11e6-8eda-d7513cf38ea5.png)

### After Patch
![template-tree4](https://cloud.githubusercontent.com/assets/2803503/22887900/b0cb00d0-f1fb-11e6-976b-ab3c90b4569d.png)

### Documentation Changes Required
None
